packages/ui-components/babel.config.js

```diff
@@ -1,12 +1 @@
-module.exports = {
-  presets: [
-    [
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current'
-        }
-      }
-    ]
-  ]
-};
+module.exports = require('@jupyterlab/testutils/lib/babel.config');
```

Comment:
This was here originally to support the storybook stuff, right? Are there any potential issues in having the storybook/test babel.config overlap like this?

Reply:
Good catch, thanks!  I'll make sure storybook is working before we consider merging this.

Reply:
I think the contents of testutils/lib/babel.config and ui-components/babel.config.js are currently identical, so it shouldn't cause an issue right now. What I was wondering is if there was any reason for them to diverge in the future. If not, then it should be fine

Reply:
I added a CI test to make sure storybook builds.